src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -245,6 +241,9 @@ else if (!type.isUnknownType()) {
         }
 
         if (paramType.isReferenceType()) {
+          // Caution: Calling .get() on an Optional without checking if it's present.
```

Comment:
Maybe this is because Specimin is using an old version of JavaParser. According to this [documentation](https://www.javadoc.io/doc/com.github.javaparser/javaparser-core/3.5.0/com/github/javaparser/resolution/types/ResolvedReferenceType.html#getTypeDeclaration--), `.getTypeDeclaration` will not return an Optional.